articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
+## 
+Synapse has two content controls: the `Sharing setting` and `Conditions for Use.` The `Sharing setting` is an access control. It determines _who_ has access to Synapse content. `Conditions for Use` are for data governance. They determine _how_ data can be used. 
 
 ## Sharing setting
-The `Sharing setting` enables you to invite others to view or collaborate on a project you have created. Projects can be **Public** visible to all or **Private** and visible to selected few. Public projects and contents are shared **broadly** with all registered Synapse users while Private projects and content are shared **sparsely**, with a limited set of designated users (e.g., a team) or not shared at all. **By default, your newly created project is private;** you are the only person that can access it or any content included in it. You can use the `Sharing setting` to enable others to view the content of your projects.
-{% include important.html content="Users are responsible for determining the appropriate Sharing Setting for any content they upload into Synapse." %}
-
-#### Create a team
-If you want to share a project with a specific group of people, you can create a team. Any Synapse user can create a team and invite other Synapse users to join their team. Only after the invited user accepts this invitation is he/she added to the team. Similarly, any Synapse user can request to be added to a team.
+You can use the `Sharing setting` to invite people to view or collaborate on a Synapse `Project`. There are two `Sharing settings`: **Public** and **Private**. Synapse content with the **Public** `Sharing setting` is visible to all Synapse users. By contrast, the **Private** `Sharing setting` limits who can see content. **By default, new `Projects` are Private.** 
+{% include important.html content="Synapse users are responsible for determining the appropriate `Sharing Setting` for any content they upload into Synapse." %}
 
 ### How to share content
-To give others access to your project and its content, click on the Share icon in the upper right hand corner of the screen. One of two icons will be displayed: either a globe, meaning the project is shared broadly (public), or a lock, meaning that the project is private and only visible to investigators you have authorized. Clicking on the lock will open a window listing the users who have access to the private project and their roles/privileges.
+To adjust the `Sharing setting`, click the `Share` button at the upper right hand side of the `Project's` header. This will open a window listing the Synapse users who have access to the `Project` and their roles. A globe icon means the `Project` is **Public**. A lock icon means the `Project` is **Private**. When you create a new `Project` you are the only Synapse user listed in this window. Use the `Add People` feature to add collaborators individually or to add a team of collaborators.
+
```

Comment:
Again, very project specific. We need information about creating/deleting local sharing settings.